components/big_cartel/big_cartel.app.mjs

```diff
@@ -1,11 +1,49 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "big_cartel",
   propDefinitions: {},
   methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    _getUrl(path) {
+      return `https://api.bigcartel.com/v1${path}`;
+    },
+    _getAccessToken() {
+      return this.$auth.oauth_access_token;
+    },
+    _getAccountId() {
+      return this.$auth.account_id;
+    },
+    _getHeaders(headers = {}) {
+      return {
+        "Authorization": `Bearer ${this._getAccessToken()}`,
+        "Accept": "application/vnd.api+json",
+        "Content-type": "application/vnd.api+json",
+        "User-Agent": "@PipedreamHQ/pipedream v0.1",
+        ...headers,
+      };
+    },
+    async _makeRequest({
+      $ = this, path, headers, ...otherConfig
```

Comment:
> I think you should include de prop `url` since you are using it from `utils.getResourcesStream` right?
Actually it is coming as `otherConfig` and overrides `url` in `config`. If I add `url` here I need to check it once more.